api/schemas/report.json

```diff
@@ -160,6 +160,11 @@
         "environment": {
             "description": "The name of the environment that was used for the puppet run (e.g. \"production\").",
             "type":        "string"
+        },
+
+        "corrective_change": {
+            "description": "True if this report suffered an unexpected change, and Puppet corrected it, or was intending to correct it if the resource or entire run was noop.",
```

Comment:
How about "True if a change or noop event was caused by an unexpected change to the system between Puppet runs"?